src/modules/addTask.js

```diff
@@ -0,0 +1,30 @@
+import { TasksClass } from './tasksClass.js';
+// import { retainCheck } from './taskStatus.js';
```

Comment:
As part of the [ruby best practices](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md) kindly remove this commented code from your codebase. Leaving commented code in your codebase is generally not a good practice. Commented-out code can clutter your codebase, make it harder to read, and can potentially lead to confusion or outdated information. If you need to review a previous version of your code, you can always use git history. :pray:

Reply:
I appreciate the reminder.